scripts/reprocess_quarantine.py

```diff
@@ -2,32 +2,36 @@
 from pika import URLParameters
 from sdc.rabbit.publishers import QueuePublisher, PublishMessageError
 
-from app import create_and_wrap_logger
-from app import settings
-
-logger = create_and_wrap_logger(__name__)
-
 
 def reprocess():
-    connection = BlockingConnection(URLParameters(settings.RABBIT_URL))
+    # To use this script, fill in the credentials for rabbit.  They should be found in app/settings.py
+    rabbit_url = 'amqp://<user>:<pass>@<host>:<port>/%2f'
```

Comment:
Why couldn't this be an environment variable?

Reply:
See other explanation ^